benchmark/shear_bands/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
 
-FIND_PACKAGE(Aspect REQUIRED HINTS ${ASPECT_DIR} ../ ../../ $ENV{ASPECT_DIR})
+FIND_PACKAGE(Aspect REQUIRED HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
```

Comment:
That created cmake error messages for me. They were of the form: ... tried to access variable name ${ASPECT_DIR} that was not defined (I can look for the exact wording if you need it).